.circleci/smoke.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+set -e
+
+TMPDIR="$(mktemp -d 2>/dev/null || mktemp -d -t 'apiejs-smoketest')"
+API_ELEMENTS="$(pwd)"
+
+create_package() {
+  cd "$TMPDIR" && npm init --yes && cd "$API_ELEMENTS"
+}
+
+install() {
+  PACKAGE="$1"
+
+  TARBALL="$(cd packages/$PACKAGE && npm pack | tail -n1)"
+  mv "packages/$PACKAGE/$TARBALL" "$TMPDIR"
+
+  cd "$TMPDIR" && npm install "$TARBALL" && cd "$API_ELEMENTS"
+}
+
+echo "Setting up Fury in $TMPDIR"
+
+create_package
+install fury
+install fury-adapter-apib-parser
+install fury-adapter-oas3-parser
+install fury-adapter-apiary-blueprint-parser
+install fury-adapter-apib-serializer
+install fury-cli
+
+cd "$TMPDIR"
+npx fury --help
+npx fury --version
+
+# Parse Apiary Blueprint
+cat << EOF | npx fury -
+HOST: http://www.google.com/
```

Comment:
```suggestion
HOST: http://www.example.com/
```

Reply:
Ah. I didn't even look at the contets of this file much (copied it from the test fixture from the subpackage).

Fixed in 9bab9f9 (I've also updated test fixture I copied it from).